src/histolab/masks.py

```diff
@@ -27,7 +27,7 @@
 from .types import Region
 from .util import (
     lazyproperty,
-    polygon_to_mask_array,
+    rectangle_to_mask_array,
```

Comment:
what do you think if we get rid of the suffix `array` ? `rectangle_to_mask` is clear enough to me even if we don't specify the return type of this function in the name itself.

Reply:
Also ... we have a well detailed doctring for the return types. What do you think?

Reply:
mmm yeah make sense, I will remove it